examples/run_Hahn_echo.py

```diff
@@ -208,6 +223,7 @@ def verifyParams():
 while save_file:
     try:
         print("SAVING FILE IN TARGET DIRECTORY...")
+        ## JF 5/6 (PR #11) → rather than using these "date" and "output_name" parameters, use the basename and counter from the config file to determine the file name
         data.hdf5_write(date+'_'+output_name+'.h5',
```

Comment:
also resolve this for this PR

Reply:
so, actually I would arrange the config file differently than I did.  I would have the following sections:
* a section called "sample" or "chemical" --> here you set a part of the filename that has the name of the chemical and the date.  You also have a default target directory
* and experiment-specific section -- this would have any additions to the basic filename that you might want (if you want them).  It would also have the default node name  it would also store the "increment" --> the increment should probably be apply to the node name, rather than to the file name (e.g. if you run multiple hahn echoes, they go into the same file with different node names)

to do this well, you  want to find out -- what happens when you ask configparser for a parameter that doesn't exist?  [this](https://stackoverflow.com/questions/6107149/how-can-i-set-default-values-for-safeconfigparser) page is worth a read, since it explains that and how you might set defaults.